### ISSUE TITLE
some problem  about  the  result(output map)

### ISSUE THREAD

**Issue by yilinchenXD** [shekkizh/FCN.tensorflow#36]
@shekkizh  when i run  FCN.py to train network  on my laptop , the  loss value from  400 drops to 3 ,but the  segmentation map is so  terrible. i can hardly  see the shape of  segmentation from output map .should  i continue to  train  network , or  there are  other factor causing the  terrible situation

**Comment by shekkizh** [comment]
Initially the predictions are random and hence you will see really high values for loss. Since most of the image is background a value of prediction of background for the whole image will on average reduce the loss considerably. This is the initial loss drop you see. I would suggest training for more epochs to see results.
It is a good practice to watch number of epochs as well as training loss while training.
